notebooks/official/model_evaluation/automl_video_classification_model_evaluation.ipynb

```diff
@@ -584,14 +527,14 @@
       "source": [
         "#### Run the training pipeline\n",
         "\n",
-        "Next, you run the job to start the training job by invoking the method `run`, with the following parameters:\n",
+        "Next, you run the job to start the training job by invoking the method run, with the following parameters:\n",
```

Comment:
Modify to: "Next, run the job to start the training job by invoking the `run` method with the following parameters:"